src/macaron/__main__.py

```diff
@@ -62,25 +72,45 @@ def analyze_slsa_levels_single(analyzer_single_args: argparse.Namespace) -> None
 
     run_config = {}
 
-    if analyzer_single_args.repo_path:
-        # Generate a config dict similar to a config read from user yaml file.
+    if analyzer_single_args.config_path:
+        # Get user config from yaml file
+        run_config = YamlLoader.load(analyzer_single_args.config_path)
+        if run_config is None:
+            sys.exit(os.EX_DATAERR)
```

Comment:
Please log an error message as well.

Reply:
Fixed in https://github.com/oracle/macaron/pull/401/commits/f66b0be2bf3c2556eed5885315d39cd66b792306